.github/workflows/continuous-integration.yml

```diff
@@ -116,13 +116,21 @@ jobs:
           - "3@dev"
         postgres-version:
           - "15"
+        extension:
+          - pdo_pgsql
         include:
           - php-version: "8.0"
             dbal-version: "2.13"
             postgres-version: "14"
+            extension: pdo_pgsql
           - php-version: "8.2"
             dbal-version: "default"
             postgres-version: "9.6"
+            extension: pdo_pgsql
+          - php-version: "8.2"
+            dbal-version: "3@dev"
```

Comment:
Let's wait for a release of DBAL and use a stable version? That way you're unlikely to break the ORM build for nothing, and there is no need to make another PR to alter this.

Reply:
All right.